releasenotes/notes/add_none_propagation_style-8ee2cc7f94c1034a.yaml

```
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    propagation: This introduces ``none`` as a supported propagator for trace context extraction and injection. 
```

nit: I think the scope should be tracing:

```suggestion
    tracing: This introduces ``none`` as a supported propagator for trace context extraction and injection. 
```

---

why tracing and not propagation?

---

I was going off of this list: https://github.com/DataDog/dd-trace-py/blob/1.x/docs/contributing.rst#scope. Setting the scope to "propagation" might be confusing to users since it's a bit ambiguous but I guess the description clears this up. 

No strong feelings here. Feel free to resolve.

---

ah that's great, sorry I missed it as a reference. Thanks I'll make the change